<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: transparent;
	background: transparent;
	padding: 0;
	margin: 0;
	color: #fff;
}
#centralwidget{
	background-color: ;
	background-image: url(:/images/R4spkB.png);
}

#widget{
	background-color: rgb(9, 27, 68);
	border-radius: 20px;
}

QLineEdit{
	background-color: rgb(9, 10, 37);
	padding: 3px 5px;
	border-radius : 5px;
}

QPushButton{
	background-color: rgb(9, 10, 37);
	padding: 10px 5px;
	border-radius: 5px;
}

#to_login, #to_register{
	background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>450</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget_welcome">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="register_page">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="TugasCaslab/resources.qrc">:/icons/fugu_icon/icons/user--plus.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Sign Up</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Enter Your Information Below</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="line_email_signup">
                <property name="placeholderText">
                 <string>Email Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_pw_signup">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_confirm_pw">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Confirm Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_confirm">
                <property name="text">
                 <string/>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_signup">
             <property name="text">
              <string>Show Password</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="button_signup">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
	background-color: rgb(5, 23, 64);
}</string>
             </property>
             <property name="text">
              <string>Register</string>
             </property>
             <property name="icon">
              <iconset resource="TugasCaslab/resources.qrc">
               <normaloff>:/icons/fugu_icon/icons-shadowless/door-open-out.png</normaloff>:/icons/fugu_icon/icons-shadowless/door-open-out.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="to_login">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
	background-color: rgb(4, 22, 63);
}</string>
             </property>
             <property name="text">
              <string>Already Registered? Log in</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="login_page">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="TugasCaslab/resources.qrc">:/icons/fugu_icon/icons/user--arrow.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Log In</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Enter Your Information Below</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>27</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLineEdit" name="line_email">
                <property name="placeholderText">
                 <string>Email Address</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_pw">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_confirm2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_login">
             <property name="text">
              <string>Show Password</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="button_login">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
	background-color: rgb(5, 23, 64);
}</string>
             </property>
             <property name="text">
              <string>Log In</string>
             </property>
             <property name="icon">
              <iconset resource="TugasCaslab/resources.qrc">
               <normaloff>:/icons/fugu_icon/icons-shadowless/door-open-out.png</normaloff>:/icons/fugu_icon/icons-shadowless/door-open-out.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="to_register">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
	background-color: rgb(4, 22, 63);
}</string>
             </property>
             <property name="text">
              <string>Not Registered? Register</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="TugasCaslab/resources.qrc"/>
 </resources>
 <connections/>
</ui>
